etl-pipeline/processes/util/pdf_page.py

```diff
@@ -0,0 +1,173 @@
+import botocore.exceptions
+import multiprocessing as mp
+import pathlib
+import tempfile
+import time
+
+import checksum
+import mets_parser
+import s3
```

Comment:
I think the s3 manager in the PDF pipeline is a bit different so we might have to port that over as well